system/ee/ExpressionEngine/Service/Validation/Rule/PasswordMatchesSecurityPolicy.php

```diff
@@ -0,0 +1,45 @@
+<?php
+/**
+ * This source file is part of the open source project
+ * ExpressionEngine (https://expressionengine.com)
+ *
+ * @link      https://expressionengine.com/
+ * @copyright Copyright (c) 2003-2021, Packet Tide, LLC (https://www.packettide.com)
+ * @license   https://expressionengine.com/license Licensed under Apache License, Version 2.0
+ */
+
+namespace ExpressionEngine\Service\Validation\Rule;
+
+use ExpressionEngine\Service\Validation\ValidationRule;
+
+/**
+ * Password Validation Rule
+ */
+class PasswordMatchesSecurityPolicy extends ValidationRule
+{
+    public function validate($key, $value)
+    {
+        switch (ee()->config->item('password_security_policy')) {
+            case 'n':
+                return true;
+                break;
+            case 's':
+                return ee('Member')->calculatePasswordComplexity($value) >= 60;
+                break;
+            case 'y':
+            case 'g':
+            default:
+                return ee('Member')->calculatePasswordComplexity($value) >= 40;
+                break;
+        }
```

Comment:
Just to make sure.. the single characters match whats used elsewhere? @intoeetive

Reply:
I ask because in this file it looks like they have changed... https://github.com/ExpressionEngine/ExpressionEngine/pull/1529/files#diff-af1051c66cc328e3cf1b9d76141062401eace8e41651bd6fc2accde6bf17f30eR178

Reply:
This has been updated